resources/application-connector/values.yaml

```diff
@@ -1,4 +1,5 @@
 global:
+  applicationManagementDisabled: false
```

Comment:
I would suggest to rename this variable to applicationManagementEnabled with default true. There are other flags for faros (e.g. flag for disabling connector-service(please see please see https://github.com/kyma-project/kyma/blob/master/resources/application-connector/requirements.yaml)) that use positive logic. It would make more sense to have it consistent as it makes reasoning easier. Note: you could use the similar form of the conditions as in requirements.yaml (applicationmanagement.enabled).

Reply:
done